Dockerfile

```diff
@@ -29,4 +29,5 @@ COPY ./play \
     /app/
 
 EXPOSE 8000
-ENTRYPOINT [ "/app/entrypoint.sh" ]
```

Comment:
`ENTRYPOINT` doesn't allow you to override using a `command` in docker-compose.yml, so this enables overriding for dev workflows.

Reply:
you should be able to with `entrypoint:` according to the docs https://docs.docker.com/compose/compose-file/. My concern here is the gke deployment will stop working with this change